README.md

```diff
@@ -145,7 +151,17 @@ curl -X POST http://localhost:11434/api/generate -d '{
 }'
 ```
 
-## Tools using Ollama
+<hr/>
+
+> The Rest API is currently designed to work on localhost (since CORS is implemented for localhost only).
+> However, you can change the host and port by using the environment variables: `OLLAMA_HOST` and `OLLAMA_PORT`.
+> Example: `OLLAMA_HOST=127.0.0.1 OLLAMA_PORT=11435 ollama serve`.
+> Important note: The API server is ***unencrypted***! So if you want to access the REST API remotely, consider using an SSH-tunnel. In no case you want to publicly expose the HTTP API.
+> For developers: If you run into issues with CORS, you can disable it by starting Google Chrome with the `--disable-web-security` flag (if you are not a developer, this is not for you).
```

Comment:
I think we're okay with this now that we have `OLLAMA_ORIGINS` environment variable. However, let's move that to `troubleshooting.md` under a section "CORS errors" since we still need to consider a better experience for the CORS issues than `OLLAMA_ORIGINS`